Tasks/UseRubyVersionV0/task.json

```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 179,
+        "Minor": 181,
```

Comment:
This task should be merged on [sprint 182](https://github.com/microsoft/build-task-team/milestone/7)

Reply:
Bumped task version